From cb6d4e5df7d5b47d835365c62ef9a0d632d2e3b7 Mon Sep 17 00:00:00 2001
From: "Chen, Yu" <yu.y.chen@intel.com>
Date: Wed, 4 Apr 2018 07:50:53 +0800
Subject: [PATCH 1/5] add access libgui for vendor module

In Android P, libgui is not allowed access by vendor module

Change-Id: Ib3ac5f14d6b3cf3c0d9ddba90a09e92f0dccaa81
Signed-off-by: Chen, Yu <yu.y.chen@intel.com>
---
 rootdir/etc/ld.config.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/etc/ld.config.txt b/rootdir/etc/ld.config.txt
index 84b308d0e..9722851ac 100644
--- a/rootdir/etc/ld.config.txt
+++ b/rootdir/etc/ld.config.txt
@@ -428,7 +428,7 @@ namespace.default.link.system.shared_libs = %LLNDK_LIBRARIES%
 namespace.default.link.vndk_in_system.shared_libs = %VNDK_USING_CORE_VARIANT_LIBRARIES%
 namespace.default.link.vndk.shared_libs  = %VNDK_SAMEPROCESS_LIBRARIES%
 namespace.default.link.vndk.shared_libs += %VNDK_CORE_LIBRARIES%
-
+namespace.default.link.vndk.shared_libs += libgui.so
 ###############################################################################
 # "runtime" APEX namespace
 #
--
2.21.0
